client/blocks/signup-form/style.scss

```diff
@@ -1,6 +1,7 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 @import "@automattic/onboarding/styles/mixins";
+@import "../../assets/stylesheets/shared/mixins/breakpoints";
```

Comment:
Storybook build of this individual file fails without explicit imports for this dependency